.travis.yml

```diff
@@ -1,13 +1,10 @@
 rvm:
-  - ruby-2.3.5-p0
+  - ruby-2.3.6-p0
 
 sudo: required
 services:
   - docker
 
-before_install:
```

Comment:
So we're now just using the default Travis version of bundler? I know we had issue doing this with WA do we really want to do this again?